tests/geometry/functional/test_so3.py

```diff
@@ -30,7 +43,7 @@ def test_adjoint(batch_size: int, dtype: torch.dtype):
     group = so3.rand(batch_size, generator=rng, dtype=dtype)
 
     # check analytic backward for the operator
-    check_lie_group_function(so3, "adjoint", TEST_EPS, group)
+    check_lie_group_function(so3, "adjoint", TEST_EPS, (group,))
```

Comment:
Minor point, but is there a workaround so you don't need to add `(x,)` for a single `x`?

Reply:
I think this is fine, since this is only used for tests.